homeassistant/components/binary_sensor/deconz.py

```diff
@@ -113,3 +113,21 @@ def device_state_attributes(self):
         if self._sensor.type in PRESENCE and self._sensor.dark is not None:
             attr[ATTR_DARK] = self._sensor.dark
         return attr
+
+    @property
+    def device(self):
+        """Description for device registry."""
+        if (self._sensor.uniqueid is not None and
+                self._sensor.uniqueid.count(':') == 7):
+            serial = self._sensor.uniqueid.split('-', 1)[0]
+        else:
+            return None
+        dev = {
+            'identifiers': [['serial', serial]],
+            'manufacturer': self._sensor.manufacturer,
+            'model': self._sensor.modelid,
+            'connection': [['Zigbee', serial]],
```

Comment:
I think that we need to extract 'zigbee' and 'serial' into constants and store them in `helpers/device_registry.py`. 

Also, I thought Zigbee had mac addresses to identify them on the network ?

Reply:
I can create the constants for the once in this PR.

Yes, it is just that they name it serial in deconz. I also need to filter out the extra characters that define each sensor on device. I can rename it Mac if you want